src/Knp/Component/Pager/Paginator.php

```diff
@@ -112,15 +112,15 @@ public function paginate($target, $page = 1, $limit = 10, array $options = array
         $itemsEvent->target = &$target;
         $this->eventDispatcher->dispatch('knp_pager.items', $itemsEvent);
         if (!$itemsEvent->isPropagationStopped()) {
-            throw new \RuntimeException('One of listeners must count and slice given target');
+            throw new \RuntimeException('One of the listeners must count and slice a given target. If this does not happen propagation is stopped.');
```

Comment:
Actually, it's quite the opposite: it fails if the event propagation isn't stopped. Thus you would rather say that: `one of the listener must count and slice a target, and stop event propagation.`, or something like that.